pkg/authenticator/config/config.go

```diff
@@ -99,11 +101,10 @@ func populateConfig() (*Config, error) {
 	}
 
 	defaultConfig := &Config{
```

Comment:
I know you didn't change this, but in light of your other changes it seems odd to me that we call this var `defaultConfig` when it literally is instantiated with the custom config the user might set in the env